example-streamer-implementations/README.md

```diff
@@ -88,4 +95,59 @@ Depending on the setup you want to test, chose any of these examples:
 | mqtt_publisher  | zenoh_subscriber |
 | mqtt_subscriber | zenoh_publisher  |
 
-The service and client will run forever. Every second a new request message is sent from the client via zenoh. That zenoh message is caught and routed over MQTT to the service. The response to the request makes the same journey in reverse.
+The service and client will run forever. Every second a new request message is sent from the client via zenoh. That Zenoh message is caught and routed over MQTT to the service. The response to the request makes the same journey in reverse.
+
+## Running the Streamer in a system consisting of an ECU that supports Zenoh and a mechatronics component using SOME/IP
+
+### Running the `zenoh_someip` binary as the Streamer instance
+
+The other available reference implementation streams between the Zenoh and SOME/IP transports.
+
+For this implementation you need to add the path to your vsomeip library to the environment, for example by setting it in your .cargo/config.toml:
+
+```toml
+[env]
+VSOMEIP_INSTALL_PATH=<path to your vsomeip lib (for example /usr/local/lib)>
+```
+
+Run the "zenoh_someip" binary of the streamer with the default configuration file:
+
+```bash
+cargo run -bin zenoh_someip --features="zenoh-transport vsomeip-transport" -- --config="DEFAULT_CONFIG.json5"
```

Comment:
```suggestion
cargo run -bin zenoh_someip --features="zenoh-transport vsomeip-transport bundled-someip" -- --config="DEFAULT_CONFIG.json5"
```

Could also note the ability to include the `bundled-vsomeip` feature flag to avoid needing to compile your own vsomeip.

Let's do this in a follow-up PR so we can merge what you've done so far! ❤️